README.md

```diff
@@ -506,6 +506,14 @@ Browsershot also can get the body of an html page after JavaScript has been exec
 Browsershot::url('https://example.com')->bodyHtml(); // returns the html of the body
 ```
 
+### COOKIE
```

Comment:
Please move this somewhere lower in the readme with the rest of the regular function and refrain form using all caps in the title.

Reply:
I styled it like the other Output Options in the Readme ... it's HTML, it's PDF ... so I named it accordingly also in CAPS ... COOKIE ... sure you want this one in another Style?

Reply:
HTML and PDF are acronyms, so they should be capitalized in any usage except file extensions. Cookie is a word, not an acronym, and so it shouldn't be in all caps unless other non-acronyms are also in all caps, which isn't the case here.

I'm not sure Cookie is generally considered an output format, though. I mean, I see how it might be, but cookies aren't the content of the page, but rather metadata for it, and metadata is extracted separately from content. (Hence the preference for `getCookie()` elsewhere.)